docs/source/transaction_freeze_ledgers.md

```diff
@@ -2,14 +2,33 @@
 
 #### Why It May Be Needed
 
-- Removing tokens need of mechanism of painless way. Transaction freeze ledgers give a security transition. Transaction Freeze Ledgers disable specific ledgers (system ignore frozen ledgers).
+- Removing of deprecated ledgers need of painless mechanism. Transaction freeze ledgers give a security transition. Transaction Freeze Ledgers disable specific ledgers (system ignore frozen ledgers).
+Some ledgers can become deprecated and outdated. By means of auth map we could prohibit actions with these ledgers. Cons of this approach:
+- it will not be obvious to the user that the ledger is out of date and why the transactions stopped being recorded
+- the ledger can be returned to work, which in view of the long downtime can cause technical and business issues.
+- the ledger cannot be removed from the system and will require to spend money on resource allocation and support of the outdated database.
+To get rid of these drawbacks, a ledger freeze transaction was created. It allows to completely freeze the ledger and remove it depending on the steward's desire.
```

Comment:
Deleting a frozen ledger that contains transactions has an adverse affect on being able to audit history.  Refer to https://github.com/hyperledger/indy-node/issues/1659 for additional details.

The ramifications of deleting a frozen ledger should be clearly documented.

Reply:
I added a note about this.